examples/python/get_open_access_pdf/README.md

```diff
@@ -0,0 +1,18 @@
+# Get Open Access PDFs
+
+This example downloads PDFs for [Open Access](https://www.openaccess.nl/en/what-is-open-access) papers by `paper_id`.
+
+Optional: Edit `.env` and set `S2_API_KEY` to your Semantic Scholar API key.
+
+To run the example locally:
+
+```sh
+python3 -m pip install -Ur requirements.txt
+python3 basic.py
+```
+
+To run the example in a Docker container:
+
+```sh
+./run_docker.sh basic.py
+```
```

Comment:
can you say a little bit about the difference between basic.py and advanced.py?

Reply:
`basic.py` uses the `requests` module from the standard library for compatibility and readability.
`advanced.py` uses the `aiohttp` and `asyncio` packages and asynchronous Python for parallel downloads.

Reply:
I tried to teach best practices while staying beginner friendly.